github/helpers/githubSDK.ts

```diff
@@ -496,76 +496,92 @@ export async function getUserAssignedIssues(
             url,
         );
 
-        const getAssignees = (assignees : any[]) : string[] => assignees.map((val): string => {
+        const getAssignees = (assignees: any[]): string[] => assignees.map((val): string => {
             return val.login as string;
         })
 
-        const modifiedResponse : Array<IGitHubIssue> = response.items.map((value) : IGitHubIssue => {
+        const modifiedResponse: Array<IGitHubIssue> = response.items.map((value): IGitHubIssue => {
             return {
-                issue_id : value.id as string,
-                issue_compact : value.body as string,
-                repo_url : value.repository_url as string,
-                user_login : value.user.login as string,
-                user_avatar : value.user.avatar_url as string,
-                number : value.number as number,
-                title : value.title as string,
-                body : value.body as string,
-                assignees : getAssignees(value.assignees),
-                state : value.state as string,
-                last_updated_at : value.updated_at as string,
-                comments : value.comments as number,
+                issue_id: value.id as string,
+                issue_compact: value.body as string,
+                repo_url: value.repository_url as string,
+                user_login: value.user.login as string,
+                user_avatar: value.user.avatar_url as string,
+                number: value.number as number,
+                title: value.title as string,
+                body: value.body as string,
+                assignees: getAssignees(value.assignees),
+                state: value.state as string,
+                last_updated_at: value.updated_at as string,
+                comments: value.comments as number,
             }
         })
 
         return modifiedResponse;
     }
-    catch(e){
+    catch (e) {
         return [];
     }
 }
 
 export async function getIssueData(
-    repoInfo:String,
-    issueNumber:String,
-    access_token:String,
-    http:IHttp
-) : Promise<IGitHubIssue> {
+    repoInfo: String,
+    issueNumber: String,
+    access_token: String,
+    http: IHttp
+): Promise<IGitHubIssue> {
     try {
-        const response = await getRequest(http, access_token, BaseRepoApiHost + repoInfo + '/issues/' + issueNumber);
-        const getAssignees = (assignees : any[]) : string[] => assignees.map((val): string => {
+        let result = await http.get(
+            `https://api.github.com/repos/${repoInfo}/issues/${issueNumber}`,
+            {
+                headers: {
+                    "Content-Type": "application/json",
+                },
+            }
+        );
+
+        // If it isn't a 2xx code, something wrong happened
+        if (!result.statusCode.toString().startsWith("2")) {
+            throw result;
+        }
```

Comment:
@ayush3160 This change limits the functionality of the `getIssueData` method. This code will not fetch data for any private repos as you have removed the access token from the http request. This should be conditionally handled for cases where the access token exists.